src/index.html

```diff
@@ -13,10 +13,119 @@
     <title>Moyo header</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="reset.css"
+    />
+    <link
+      rel="stylesheet"
+      href="style.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="main_logo"
+        title="On main page"
+      >
+        <img
+          class="logo"
+          src="/src/images/logo.png"
+          width="40"
+          height="40"
+          alt="Moyo.png"
```

Comment:
The alt attribute for the image is 'Moyo.png', which is not descriptive. Alt text should describe the content or function of the image, such as 'Moyo Logo'.